### PR TITLE
Add a quick note about state version values

### PR DESCRIPTION
**Pull request by nalind**
Here's that part of #146 that I didn't get in fast enough.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ 79b0610

File: buildah.go [hunk 1]
```diff
@@ -19,9 +19,17 @@ const (
...
 	Package = "buildah"
 	// Version for the Package
-	Version       = "0.1"
+	Version = "0.1"
+	// The value we use to identify what type of information, currently a
+	// serialized Builder structure, we are using as per-container state.
+	// This should only be changed when we make incompatible changes to
+	// that data structure, as it's used to distinguish containers which
+	// are "ours" from ones that aren't.
 	containerType = Package + " 0.0.1"
```
Comment (TomSweeneyRedHat) [inline buildah.go]: IDK as I'm flip fllopping, so I'll just ask.   Should/do we need to state the basis of the Builder structure?  i.e. based on Docker's Builder structure or whatever?  Do we want to identify who "They" is somewhere in here?  I'm getting at this as we might make incompatible changes, but isn't it likely "They" might too?

Comment (nalind) [inline buildah.go]: We're talking about the parts of the structure that we pick up from other packages (the OCIv1 configuration structure), right?  That's defined as part of the image-spec, and I guess we could stand to call it out.

Comment (TomSweeneyRedHat) [inline buildah.go]: If you get a chance to add verbiage later about the ociv1 spec here, I think it would make it clear, thx.

**Pull request closed without merging**